libmscore/CMakeLists.txt

```diff
@@ -45,7 +45,7 @@ add_library (
 
       types.h accidental.h ambitus.h arpeggio.h articulation.h audio.h bagpembell.h barline.h beam.h bend.h
       box.h bracket.h bracketItem.h breath.h bsp.h bsymbol.h changeMap.h chord.h chordline.h chordlist.h chordrest.h clef.h
-      cleflist.h connector.h drumset.h dsp.h duration.h durationtype.h dynamic.h element.h
+      cleflist.h connector.h drumset.h dsp.h duration.h durationtype.h dynamic.h easeInOut.h element.h
```

Comment:
Ah!!! It needs to be `EaseInoOut.h", uppercsase `E`, case sensitive (not sure though why it works on Mac, this has a case sentitife file system too)
Or better rename the files to start with a lower case`e`

Reply:
Done. I decided on renaming the file with a lowercase 'e'. Thanks